addons/account/tests/test_invoice_tax_totals.py

```diff
@@ -0,0 +1,411 @@
+# -*- coding: utf-8 -*-
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+import json
+
+
+@tagged('post_install', '-at_install')
+class TestTaxTotals(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.tax_group1 = cls.env['account.tax.group'].create({'name': '1'})
+        cls.tax_group2 = cls.env['account.tax.group'].create({'name': '2'})
+        cls.tax_group_after1 = cls.env['account.tax.group'].create({'name': 'after 1', 'after_total': True})
+        cls.tax_group_after2 = cls.env['account.tax.group'].create({'name': 'after 2', 'after_total': True})
+
+
```

Comment:
too much lines! :boom: Maybe you could install an extension to detect such things! :roll_eyes: